src/openstack_cloud/openstack_manager.py

```diff
@@ -794,7 +796,7 @@ def _create_runner(args: _CreateRunnerArgs) -> None:
                     security_groups=[SECURITY_GROUP_NAME],
                     userdata=cloud_userdata_str,
                     auto_ip=False,
-                    timeout=120,
+                    timeout=CREATE_SERVER_TIMEOUT,
```

Comment:
Out of curiosity: when it fails by timeout, the server creation is aborted? Or could it fail by timeout but the server is still being created and at some point, is done and ready?

Reply:
I think it will not be cancelled, this is also the reason why we have a cleanup mechanism in those cases, but perhaps @yhaliaw can confirm.